.github/workflows/cleanup-after-pr.yml

```diff
@@ -28,13 +69,10 @@ jobs:
         run: az acr login -n bitwardenprod
 
       ########## Remove Docker images ##########
-      - name: Remove the Docker image from ACR
+      - name: Remove the Docker image from bitwardenprod ACR
         env:
           REF: ${{ github.event.pull_request.head.ref }}
-          REGISTRIES: |
-            registries:
-              - bitwardenprod
-              - bitwardenqa
+          REGISTRY: bitwardenprod
```

Comment:
🌱  There are multiple references to `bitwardenprod` and some duplication, you could improve this by using a global environment variable. Example https://github.com/bitwarden/server/blob/5a5e5c50580055f3dd681cf763627b0324d73e82/.github/workflows/build.yml#L13-L14

Reply:
Suggestion addressed. Thanks, @mimartin12!